### DISCUSSION TITLE
Support Native Cosmos Stables $USK (From Kujira) and $USDC (From Noble) as payment methods for leases.

### DISCUSSION THREAD

**Discussion by Rorcual-Nodes** [akash-network/community#625] [Economics]
**Context**

Currently, the only supported forms of payment on Akash Network are the network’s native AKT coin and USDC coming from Osmosis (ie wrapped version from Axelar and wrapped again through IBC from Osmosis). This creates several frictions for providers and tenants like:

**Tenant**

- Lack of price stability. This leads to tenants having difficulty understanding their costs and planning potential spending ahead of time due to the inherent price volatility of AKT. As a long term user of Akash, keeping track of balances and estimate expenses can be a bit of a challenge for non avid users. 

- If there is an aim to attract a new wave of users, the on-boarding experience should be more down to earth and easy to handle for new Akash users.

- Acquiring, using, and holding an alternative asset. Tenants have to go through the process of acquiring and holding AKT, which is not as widely accessible or as well-understood as USDC, but still the Osmosis wrapped version comprises a couple extra steps (and possible points of failure) that would bring experience to a really low standard.

**Provider**

- Lack of price stability (again). It becomes difficult to consistently and reliably forecast revenues due to the inherent price volatility of AKT (Keeping this point exactly the same as the previous proposal as I would like to know if there is an actual demand for USDC leases)

- Holding and liquidating an alternative asset. Providers have to understand how to handle, custody, and ultimately liquidate AKT earnings in order to realize revenue and implies a selling preassure on AKT that would be dampened with the acceptance of more stable forms of payment.

Collectively, these challenges make it difficult for Akash Network to acquire tenants and providers who don’t have, don’t understand, or don’t wish to acquire/hold AKT or suffer the pains of multiple IBC and/or bridges.

I must admit I am speaking purely as an Akash Network user and not as an AKT token holder, but if some cashflow from the stable rates can end somehow on the stakers rewards or AKT buybacks it would make everyone happier. 

Stable payments aim to assuage resource planning concerns around a volatile alternative asset, making the experiences for both providers and tenants on Akash Network more consistent and accessible.

**The case for USK and USDC**

Both of the proposed assets are well known already within the Cosmos ecosystem and have proved their strength. The strongest point in favour for this is two assets is the simplicity of an IBC transaction from any of the respective chains that would allow funds to be moved quickly and seamlessly without the need of the extra step of Bridging from Ethereum or buying on Osmosis and double wrap the Ethereum already wrapped version. Some key features for each option:

- USK from Kujira: The over-collateralized soft-pegged stable from Kujira has proven stability both in the ups an down markets, continiously increasing its use-cases both within and outside the Kujira ecosystem.

  This one comes with an extra chance to increase both AKT exposure to the Cosmos (In the form of stronger LPs, on-chain Orderbook trading experience as FIN and collateral for borrowing other assets such as USK, USDC, etc.) as well as increasing the USK collateral palette since a strong bonding between the two communities can lead to AKT being available as a Mint collateral for USK.

- USDC from Noble: Although still being the new kid in the block, native USDC on Cosmos is gaining exposure and stability in the ecosystem. The seamless bridging experience from a great amount of well-known chain both within and outside the Cosmos makes it a very strong choice for stable payments.

**Last Thoughts**

To further support the above mentioned arguments, I would like to mention that my latest UX with Akash has increased by a mile with the adoption of private images and stable payments (hopefully with a reduction in frictions with the above proposal).

We are working closely with MantaDAO (POL Market Maker on Kujira) who are willing to deploy their whole stack on Akash fully decentralized and at some point hopefully managed by DAO governance (In talks with DAODAO). Further stable payment methods would be the cherry on top both for accounting and governance management.

We have also been approached by other teams willing to follow these steps so we consider the addition of these two stables a step towards supreme UX experience.

**Comment by PragmaticMonkey** [comment]
Just a note to say that at MantaDAO we would love to be able to use $USK to pay for their Akash leases. We currently run MantaSwap 100% on Akash and will deploy some more of our apps there. We are also preaching for many other projects on Kujira to do the same. Having USK as a payment option would be of great use to all of us.

**Comment by 31trainman** [comment]
I myself have proposed support for noble Usdc. I was told it was somewhere in the pipeline. Perhaps we can work together to bring Usdc on noble network to akash etc

**Reply by Rorcual-Nodes** [reply]
I assume it requires a gov proposal with the right ibc denomination, pretty much the same as current USDC option.

@gosuri Can you chime in and maybe put us in the right path?

**Comment by rodri-r** [comment]
Nice idea and it does make sense. Like 31trainman says, USDC Noble is already on the roadmap, but I am not sure when this will be added to the network. As Rorcual-Nodes mentions, this would require an on-chain governance proposal and some modifications to the network, for enabling both USK and USDC.I believe this would depend on the core team,to determine the amount of time it would take them and when they could add  this to the network, but I do like the idea.

**Comment by Rorcual-Nodes** [comment]
@rodri-r @31trainman any chance you can get any of the core team involved here? We would like to know if it's just a matter of getting a prop on chain or it actually needs some devving form the core team.

**Reply by rodri-r** [reply]
Hey, I believe this does require some devving from core team, as well as some discussion  on the topic,before it can be made into an on chain prop and decided by governance. 

Most of the core team where at ETHCC and Nebular Summmit in Brussels these past days. They should be getting back this week and see this discussion soon.

This may also need to be discussed in one or more of the working group calls.

**Reply by 31trainman** [reply]
I think we should bring this to Tyler and or the people in charge of the payment methods for deployments and see what we can get done.

**Reply by Rorcual-Nodes** [reply]
Ah yes please. Try to get involved anyone that you guys think will need to take part on it either here or on DC.

Happy to help make it as smooth as possible for everyone.

**Comment by brewsterdrinkwater** [comment]
Thank you for sharing this. As mentioned above, there are some other discussions around Noble USDC, so this is something that the Akash community is interested in.

Normally this would be a parameter change that anyone can put on chain for vote, but core engineering still needs to unlock the full functionality of packet forwarding. 

The sig-chain group talked about this discussion yesterday at the meeting. Here is the [video](https://xv63id5ly2ezaqwhzqj2hqgmcupv2tu6qfnwz6gl7pq2zuysn2tq.arweave.net/vX20D6vGiZBCx8wTo8DMFR9dTp6BW2z4y_vhrNMSbqc) with conversation starting around 8:10. Notes will be available shortly. 

Plans to unlock full functionality to make this a parameter change are expected in early fall 2024, AFTER the Cosmos SDK 47 upgrade. 

Feel free to join future sig-economics meetings to get more involved in these kind of discussions.
